src/displayapp/fonts/README.md

```diff
@@ -31,6 +31,23 @@ Add new symbols:
 static constexpr const char* newSymbol = "\xEF\x86\x85";
 ```
 
+#### Fix the zero symbol
+If you don't wish the inner dot of the 0 (zero) symbol to stick to the boundary, edit `src/displayapp/fonts/jetbrains_mono_bold_20.c` and replace:
+
+    /* U+0030 "0" */
+    0x3f, 0x1f, 0xef, 0x3f, 0x87, 0xe1, 0xf8, 0x7f,
+    0xdf, 0xf7, 0xe1, 0xf8, 0x7e, 0x1f, 0xcf, 0x7f,
+    0x8f, 0xc0,
+
+with
+
+    /* U+0030 "0" */
+    0x3f, 0x1f, 0xef, 0x3f, 0x87, 0xe1, 0xf8, 0x7e,
+    0xdf, 0xb7, 0xe1, 0xf8, 0x7e, 0x1f, 0xcf, 0x7f,
+    0x8f, 0xc0,
+
+(there are two changes: 7f -> 7e and f7 -> b7)
+
```

Comment:
I think these instructions should be moved to the "Generate the fonts" section, maybe below the line where it says to download the file. Also the instructions should be worded in a way that makes this a requirement rather than a decision. This is to make sure this fix is included each time the font is updated in InfiniTime. It could be something like "To fix an error that happens during the font conversion...".

Reply:
But it already was in the "Generate the fonts" section!
Anyway, I removed the possible misleading subheader, and reworded the text to be more imperative.